cekit/builders/osbs.py

```diff
@@ -354,7 +358,12 @@ def run(self):
                     task_id = subprocess.check_output(cmd).strip().decode("utf8")
 
                 except subprocess.CalledProcessError as ex:
-                    raise CekitError("Building container image in OSBS failed", ex)
+                    LOGGER.error(
+                        "{} Command stdout is '{}' with stderr '{}'".format(
+                            ex, ex.stdout, ex.stderr
+                        )
+                    )
+                    raise CekitError("Building container image in OSBS failed")
```

Comment:
Pros/cons here ; while we do get a nice CekitError bubbling up ; it would be equally possible to remove the try/catch and just let the CalledProcessError bubble all the way up

Reply:
I think I'll remove this particular try/catch.